### ISSUE TITLE
Conda install/activate gives INFO messages and then things are broken

### ISSUE THREAD

**Issue by kgullikson88** [conda/conda#7011]
**I'm submitting a...**
  - [x ] bug report
  - [ ] feature request

## Current Behavior

When I install a bunch of packages in a new environment and then try to activate the environment, I get a bunch of messages and then conda is broken.

### Steps to Reproduce

```
conda create -n template_testing2 pytorch imbalanced-learn mkl-service numpy scipy pandas statsmodels scikit-learn keras=2.0.6 numba theano tensorflow cython joblib dask h5py pytables hdbscan matplotlib seaborn bokeh datashader ipython ipykernel holoviews python=3.6
Solving environment: done

## Package Plan ##

  environment location: /home/kgullikson/miniconda3/envs/template_testing2

  added / updated specs:
    - bokeh
    - cython
    - dask
    - datashader
    - h5py
    - hdbscan
    - holoviews
    - imbalanced-learn
    - ipykernel
    - ipython
    - joblib
    - keras=2.0.6
    - matplotlib
    - mkl-service
    - numba
    - numpy
    - pandas
    - pytables
    - python=3.6
    - pytorch
    - scikit-learn
    - scipy
    - seaborn
    - statsmodels
    - tensorflow
    - theano


The following NEW packages will be INSTALLED:

    absl-py:                0.1.10-py36_0
    binutils_impl_linux-64: 2.28.1-h04c84fa_2
    binutils_linux-64:      7.2.0-25
    bleach:                 1.5.0-py36_0
    bokeh:                  0.12.14-py36_0
    bzip2:                  1.0.6-h9a117a8_4
    ca-certificates:        2017.08.26-h1d4fec5_0
    certifi:                2018.1.18-py36_0
    cffi:                   1.11.5-py36h9745a5d_0
    click:                  6.7-py36h5253387_0
    cloudpickle:            0.5.2-py36_1
    colorcet:               1.0.0-py36_0
    cudatoolkit:            8.0-3
    cudnn:                  7.0.5-cuda8.0_0
    cycler:                 0.10.0-py36h93f1223_0
    cython:                 0.27.3-py36h1860423_0
    dask:                   0.17.1-py36_0
    dask-core:              0.17.1-py36_0
    datashader:             0.6.2-py36h21a1380_0
    datashape:              0.5.4-py36h3ad6b5c_0
    dbus:                   1.12.2-hc3f9b76_1
    decorator:              4.2.1-py36_0
    distributed:            1.21.2-py36_0
    entrypoints:            0.2.3-py36h1aec115_2
    expat:                  2.2.5-he0dffb1_0
    fontconfig:             2.12.4-h88586e7_1
    freetype:               2.8-hab7d2ae_1
    gcc_impl_linux-64:      7.2.0-hc5ce805_2
    gcc_linux-64:           7.2.0-25
    glib:                   2.53.6-h5d9569c_2
    gmp:                    6.1.2-h6c8ec71_1
    gst-plugins-base:       1.12.4-h33fb286_0
    gstreamer:              1.12.4-hb53b477_0
    gxx_impl_linux-64:      7.2.0-hd3faf3d_2
    gxx_linux-64:           7.2.0-25
    h5py:                   2.7.1-py36h3585f63_0
    hdbscan:                0.8.12-py36_1               conda-forge
    hdf5:                   1.10.1-h9caa474_1
    heapdict:               1.0.0-py36_2
    holoviews:              1.9.4-py36_0
    html5lib:               0.9999999-py36_0
    icu:                    58.2-h9c2bf20_1
    imbalanced-learn:       0.3.3-py_0                  conda-forge
    intel-openmp:           2018.0.0-hc7b2577_8
    ipykernel:              4.8.2-py36_0
    ipython:                6.2.1-py36h88c514a_1
    ipython_genutils:       0.2.0-py36hb52b0d5_0
    ipywidgets:             7.1.2-py36_0
    jedi:                   0.11.1-py36_0
    jinja2:                 2.10-py36ha16c418_0
    joblib:                 0.11-py36_0
    jpeg:                   9b-h024ee3a_2
    jsonschema:             2.6.0-py36h006f8b5_0
    jupyter:                1.0.0-py36_4
    jupyter_client:         5.2.2-py36_0
    jupyter_console:        5.2.0-py36he59e554_1
    jupyter_core:           4.4.0-py36h7c827e3_0
    keras:                  2.0.6-py36_0                conda-forge
    kiwisolver:             1.0.1-py36h764f252_0
    libedit:                3.1-heed3624_0
    libffi:                 3.2.1-hd88cf55_4
    libgcc-ng:              7.2.0-h7cc24e2_2
    libgfortran-ng:         7.2.0-h9f7466a_2
    libgpuarray:            0.7.5-h14c3975_0
    libpng:                 1.6.34-hb9fc6fc_0
    libprotobuf:            3.5.1-h6f1eeef_0
    libsodium:              1.0.15-hf101ebd_0
    libstdcxx-ng:           7.2.0-h7a57d05_2
    libtiff:                4.0.9-h28f6b97_0
    libxcb:                 1.12-hcd93eb1_4
    libxml2:                2.9.7-h26e45fe_0
    llvmlite:               0.22.0-py36ha27ea49_0
    locket:                 0.2.0-py36h787c0ad_1
    lzo:                    2.10-h49e0be7_2
    mako:                   1.0.7-py36h0727276_0
    markdown:               2.6.11-py36_0
    markupsafe:             1.0-py36hd9260cd_1
    matplotlib:             2.2.0-py36hbc4b006_0
    mistune:                0.8.3-py36_0
    mkl:                    2018.0.1-h19d6760_4
    mkl-service:            1.1.2-py36h17a0993_4
    msgpack-python:         0.5.5-py36h6bb024c_0
    multipledispatch:       0.5.0-py36_0
    nbconvert:              5.3.1-py36hb41ffb7_0
    nbformat:               4.4.0-py36h31c9010_0
    nccl:                   1.3.4-cuda8.0_1
    ncurses:                6.0-h9df7e31_2
    notebook:               5.4.0-py36_0
    numba:                  0.37.0-np114py36h5fcf951_0
    numexpr:                2.6.4-py36hc4a3f9a_0
    numpy:                  1.14.1-py36ha266831_2
    olefile:                0.45.1-py36_0
    openssl:                1.0.2n-hb7f436b_0
    packaging:              17.1-py36_0
    pandas:                 0.20.3-py36h6022372_2
    pandoc:                 1.19.2.1-hea2e7c5_1
    pandocfilters:          1.4.2-py36ha6701b7_1
    param:                  1.5.1-py36_0
    parso:                  0.1.1-py36h35f843b_0
    partd:                  0.3.8-py36h36fd896_0
    patsy:                  0.5.0-py36_0
    pcre:                   8.41-hc27e229_1
    pexpect:                4.4.0-py36_0
    pickleshare:            0.7.4-py36h63277f8_0
    pillow:                 5.0.0-py36h3deb7b8_0
    pip:                    9.0.1-py36_5
    prompt_toolkit:         1.0.15-py36h17d85b1_0
    protobuf:               3.5.1-py36hf484d3e_0
    psutil:                 5.4.3-py36h14c3975_0
    ptyprocess:             0.5.2-py36h69acd42_0
    pycparser:              2.18-py36hf9f622e_1
    pygments:               2.2.0-py36h0d3125c_0
    pygpu:                  0.7.5-py36h14c3975_0
    pyparsing:              2.2.0-py36hee85983_1
    pyqt:                   5.6.0-py36h0386399_5
    pytables:               3.4.2-py36h3b5282a_2
    python:                 3.6.4-hc3d631a_1
    python-dateutil:        2.6.1-py36h88d3b88_1
    pytorch:                0.3.0-py36cuda8.0cudnn7.0_0
    pytz:                   2018.3-py36_0
    pyyaml:                 3.12-py36hafb9ca4_1
    pyzmq:                  17.0.0-py36h14c3975_0
    qt:                     5.6.2-hd25b39d_14
    qtconsole:              4.3.1-py36h8f73b5b_0
    readline:               7.0-ha6073c6_4
    scikit-learn:           0.19.1-py36h7aa7ec6_0
    scipy:                  1.0.0-py36hbf646e7_0
    seaborn:                0.8.1-py36hfad7ec4_0
    send2trash:             1.5.0-py36_0
    setuptools:             38.5.1-py36_0
    simplegeneric:          0.8.1-py36_2
    sip:                    4.18.1-py36h51ed4ed_2
    six:                    1.11.0-py36h372c433_1
    sortedcontainers:       1.5.9-py36_0
    sqlite:                 3.22.0-h1bed415_0
    statsmodels:            0.8.0-py36h8533d0b_0
    tblib:                  1.3.2-py36h34cf8b6_0
    tensorflow:             1.5.0-0
    tensorflow-base:        1.5.0-py36hff88cb2_1
    tensorflow-tensorboard: 1.5.1-py36hf484d3e_0
    terminado:              0.8.1-py36_1
    testpath:               0.3.1-py36h8cadb63_0
    theano:                 1.0.1-py36h6bb024c_0
    tk:                     8.6.7-hc745277_3
    toolz:                  0.9.0-py36_0
    tornado:                5.0-py36_0
    traitlets:              4.3.2-py36h674d592_0
    wcwidth:                0.1.7-py36hdf4376a_0
    werkzeug:               0.14.1-py36_0
    wheel:                  0.30.0-py36hfd4bba0_1
    widgetsnbextension:     3.1.4-py36_0
    xarray:                 0.10.1-py36_0
    xz:                     5.2.3-h55aa19d_2
    yaml:                   0.1.7-had09818_2
    zeromq:                 4.2.3-h439df22_3
    zict:                   0.1.3-py36h3a3bf81_0
    zlib:                   1.2.11-ha838bed_2

Proceed ([y]/n)? y

Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate template_testing2
#
# To deactivate an active environment, use
#
#     $ conda deactivate
```
That seems to work, however, when I activate:
```
conda activate template_testing2

INFO: activate-binutils_linux-64.sh made the following environmental changes:
+ADDR2LINE=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-addr2line
+AR=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-ar
+AS=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-as
+CXXFILT=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-c++filt
+ELFEDIT=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-elfedit
+GPROF=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-gprof
+HOST=x86_64-conda_cos6-linux-gnu
+LD_GOLD=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-ld.gold

▽
+LD=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-ld
+NM=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-nm
+OBJCOPY=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-objcopy
+OBJDUMP=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-objdump
+RANLIB=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-ranlib
+READELF=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-readelf
+SIZE=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-size
+STRINGS=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-strings
+STRIP=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-strip
INFO: activate-gcc_linux-64.sh made the following environmental changes:
+CC=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-cc
+CFLAGS=-march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -pipe
+CPPFLAGS=-DNDEBUG -D_FORTIFY_SOURCE=2 -O2
+CPP=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-cpp
+DEBUG_CFLAGS=-march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-all -fno-plt -Og -g -Wall -Wextra -fvar-tracking-assignments -pipe
+DEBUG_CPPFLAGS=-D_DEBUG -D_FORTIFY_SOURCE=2 -Og
+GCC_AR=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-gcc-ar
+GCC=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-gcc
+GCC_NM=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-gcc-nm
+GCC_RANLIB=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-gcc-ranlib
+LDFLAGS=-Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now
+_PYTHON_SYSCONFIGDATA_NAME=_sysconfigdata_x86_64_conda_cos6_linux_gnu
INFO: activate-gxx_linux-64.sh made the following environmental changes:
+CXXFLAGS=-fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -pipe
+CXX=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-c++
+DEBUG_CXXFLAGS=-fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-all -fno-plt -Og -g -Wall -Wextra -fvar-tracking-assignments -pipe
+GXX=/home/kgullikson/miniconda3/envs/template_testing2/bin/x86_64-conda_cos6-linux-gnu-g++
```
Just messages is fine with me, but now conda is broke:
```
(template_testing2) kgullikson@conway:~$ conda
Failed to import the site module
Traceback (most recent call last):
  File "/home/kgullikson/miniconda3/lib/python3.6/site.py", line 548, in <module>
    main()
  File "/home/kgullikson/miniconda3/lib/python3.6/site.py", line 530, in main
    known_paths = addusersitepackages(known_paths)
  File "/home/kgullikson/miniconda3/lib/python3.6/site.py", line 282, in addusersitepackages
    user_site = getusersitepackages()
  File "/home/kgullikson/miniconda3/lib/python3.6/site.py", line 258, in getusersitepackages
    user_base = getuserbase() # this will also set USER_BASE
  File "/home/kgullikson/miniconda3/lib/python3.6/site.py", line 248, in getuserbase
    USER_BASE = get_config_var('userbase')
  File "/home/kgullikson/miniconda3/lib/python3.6/sysconfig.py", line 601, in get_config_var
    return get_config_vars().get(name)
  File "/home/kgullikson/miniconda3/lib/python3.6/sysconfig.py", line 550, in get_config_vars
    _init_posix(_CONFIG_VARS)
  File "/home/kgullikson/miniconda3/lib/python3.6/sysconfig.py", line 421, in _init_posix
    _temp = __import__(name, globals(), locals(), ['build_time_vars'], 0)
ModuleNotFoundError: No module named '_sysconfigdata_x86_64_conda_cos6_linux_gnu'
```

## Expected Behavior

Activating the environment should leave everything intact?

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

Note - I had to open a new terminal to run this since conda was broken by the above.
```
     active environment : base
    active env location : /home/kgullikson/miniconda3
            shell level : 1
       user config file : /home/kgullikson/.condarc
 populated config files : /home/kgullikson/.condarc
          conda version : 4.4.11
    conda-build version : not installed
         python version : 3.6.0.final.0
       base environment : /home/kgullikson/miniconda3  (writable)
           channel URLs : https://repo.continuum.io/pkgs/main/linux-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/linux-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/linux-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/linux-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://conda.anaconda.org/kgullikson/linux-64
                          https://conda.anaconda.org/kgullikson/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/kgullikson/miniconda3/pkgs
                          /home/kgullikson/.conda/pkgs
       envs directories : /home/kgullikson/miniconda3/envs
                          /home/kgullikson/.conda/envs
               platform : linux-64
             user-agent : conda/4.4.11 requests/2.12.4 CPython/3.6.0 Linux/3.10.0-514.21.1.el7.x86_64 centos/7 glibc/2.17
                UID:GID : 1005:1005
             netrc file : None
           offline mode : False
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
==> /home/kgullikson/.condarc <==
channels:
  - defaults
  - kgullikson
  - conda-forge
```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
# packages in environment at /home/kgullikson/miniconda3:
#
# Name                    Version                   Build  Channel
bleach                    1.5.0                    py36_0    defaults
boeing-lib                0.1.0                     <pip>
bokeh                     0.12.4                   py36_0    defaults
ca-certificates           2017.08.26           h1d4fec5_0    defaults
certifi                   2018.1.18                py36_0    defaults
cffi                      1.9.1                    py36_0    defaults
chest                     0.2.3                    py36_0    defaults
click                     6.7                       <pip>
cloudpickle               0.2.2                    py36_0    defaults
conda                     4.4.11                   py36_0    defaults
conda-env                 2.6.0                h36134e3_1    defaults
corner                    2.0.1                     <pip>
cryptography              1.7.1                    py36_0    defaults
CVXcanon                  0.1.1                     <pip>
cvxpy                     0.4.9                     <pip>
cycler                    0.10.0                   py36_0    defaults
cython                    0.25.2                   py36_0    defaults
dask                      0.14.1                   py36_0    defaults
datashader                0.4.0                    py36_0    defaults
datashape                 0.5.4                    py36_0    defaults
dbus                      1.10.10                       0    defaults
decorator                 4.0.11                   py36_0    defaults
dill                      0.2.6                     <pip>
ds-tools                  0.13.0.dev0               <pip>
ecos                      2.0.4                     <pip>
entrypoints               0.2.2                    py36_1    defaults
environment-kernels       1.1                       <pip>
expat                     2.1.0                         0    defaults
fastcache                 1.0.2                     <pip>
fastcluster               1.1.22                    <pip>
fontconfig                2.12.1                        3    defaults
freetype                  2.5.5                         2    defaults
glib                      2.50.2                        1    defaults
gmp                       6.1.0                         0    defaults
gst-plugins-base          1.8.0                         0    defaults
gstreamer                 1.8.0                         0    defaults
h5py                      2.7.0               np112py36_0    defaults
hdbscan                   0.8.10                    <pip>
hdbscan                   0.8.8                     <pip>
hdf5                      1.8.17                        1    defaults
heapdict                  1.0.0                    py36_1    defaults
html5lib                  0.999                    py36_0    defaults
icu                       54.1                          0    defaults
idna                      2.2                      py36_0    defaults
imbalanced-learn          0.2.1                     <pip>
imblearn                  0.0                       <pip>
ipykernel                 4.5.2                    py36_0    defaults
ipython                   6.1.0                    py36_0    defaults
ipython_genutils          0.1.0                    py36_0    defaults
ipywidgets                7.1.2                    py36_0    defaults
jbig                      2.1                           0    defaults
jedi                      0.10.2                   py36_2    defaults
jinja2                    2.9.5                    py36_0    defaults
joblib                    0.9.4                    py36_0    defaults
joblib                    0.10.2                    <pip>
jpeg                      9b                            0    defaults
jsonschema                2.5.1                    py36_0    defaults
jupyter                   1.0.0                    py36_3    defaults
jupyter_client            5.0.0                    py36_0    defaults
jupyter_console           5.1.0                    py36_0    defaults
jupyter_core              4.3.0                    py36_0    defaults
jupyterlab                0.31.10                  py36_0    defaults
jupyterlab_launcher       0.10.4                   py36_0    conda-forge
Keras                     2.0.6                     <pip>
libffi                    3.2.1                         1    defaults
libgcc                    5.2.0                         0    defaults
libgcc-ng                 7.2.0                h7cc24e2_2    defaults
libgfortran               3.0.0                         1    defaults
libgpuarray               0.6.4                         0    defaults
libiconv                  1.14                          0    defaults
libpng                    1.6.27                        0    defaults
libpq                     9.5.4                         0    defaults
libprotobuf               3.2.0                         0    defaults
libsodium                 1.0.10                        0    defaults
libstdcxx-ng              7.2.0                h7a57d05_2    defaults
libtiff                   4.0.6                         3    defaults
libxcb                    1.12                          1    defaults
libxml2                   2.9.4                         0    defaults
lime                      0.1.1.18                 py36_0    conda-forge
line-profiler             2.0                       <pip>
llvmlite                  0.16.0                   py36_0    defaults
locket                    0.2.0                    py36_1    defaults
mako                      1.0.6                    py36_0    defaults
markupsafe                0.23                     py36_2    defaults
matplotlib                2.0.0               np112py36_0    defaults
mistune                   0.7.4                    py36_0    defaults
mkl                       2017.0.1                      0    defaults
mkl-service               1.1.2                    py36_3    defaults
multipledispatch          0.4.9                    py36_0    defaults
multiprocess              0.70.5                    <pip>
natgas                    1.0                       <pip>
nbconvert                 5.1.1                    py36_0    defaults
nbformat                  4.3.0                    py36_0    defaults
networkx                  1.11                     py36_0    defaults
nodejs                    8.9.3                h439df22_0    defaults
nose                      1.3.7                    py36_1    defaults
notebook                  5.0.0                    py36_0    defaults
numba                     0.31.0              np112py36_0    defaults
numexpr                   2.6.2               np112py36_0    defaults
numpy                     1.12.1                   py36_0    defaults
odo                       0.5.0                    py36_1    defaults
olefile                   0.44                     py36_0    defaults
openssl                   1.0.2n               hb7f436b_0    defaults
pandas                    0.19.2              np112py36_1    defaults
pandoc                    1.15.0.6                      0    defaults
pandocfilters             1.4.1                    py36_0    defaults
partd                     0.3.7                    py36_0    defaults
path.py                   10.1                     py36_0    defaults
patsy                     0.4.1                    py36_0    defaults
pcre                      8.39                          1    defaults
pexpect                   4.2.1                    py36_0    defaults
phased-lstm-keras         1.0.0                     <pip>
pickleshare               0.7.4                    py36_0    defaults
pillow                    4.1.0                    py36_0    defaults
pip                       9.0.1                    py36_1    defaults
plac                      0.9.6                     <pip>
prompt_toolkit            1.0.13                   py36_0    defaults
protobuf                  3.2.0                    py36_0    defaults
psutil                    5.2.2                    py36_0    defaults
psycopg2                  2.7.1                    py36_0    defaults
ptyprocess                0.5.1                    py36_0    defaults
pyasn1                    0.1.9                    py36_0    defaults
pycosat                   0.6.3            py36h0a5515d_0    defaults
pycparser                 2.17                     py36_0    defaults
pygments                  2.2.0                    py36_0    defaults
pygpu                     0.6.4                    py36_1    defaults
pyopenssl                 16.2.0                   py36_0    defaults
pyparsing                 2.1.4                    py36_0    defaults
pyqt                      5.6.0                    py36_2    defaults
pytables                  3.3.0               np112py36_0    defaults
python                    3.6.0                         0    defaults
python-dateutil           2.6.0                    py36_0    defaults
pytz                      2016.10                  py36_0    defaults
PyYAML                    3.12                      <pip>
pyyaml                    3.12                     py36_0    defaults
pyzmq                     16.0.2                   py36_0    defaults
qt                        5.6.2                         3    defaults
qtconsole                 4.2.1                    py36_1    defaults
readline                  6.2                           2    defaults
requests                  2.12.4                   py36_0    defaults
ruamel_yaml               0.11.14                  py36_1    defaults
scikit-learn              0.18.1              np112py36_1    defaults
scipy                     0.19.0              np112py36_0    defaults
scs                       1.2.6                     <pip>
seaborn                   0.7.1                    py36_0    defaults
setuptools                38.5.1                   py36_0    defaults
simplegeneric             0.8.1                    py36_1    defaults
sip                       4.18                     py36_0    defaults
six                       1.10.0                   py36_0    defaults
sqlalchemy                1.1.6                    py36_0    defaults
sqlite                    3.13.0                        0    defaults
statsmodels               0.8.0               np112py36_0    defaults
tensorflow                1.1.0               np112py36_0    defaults
terminado                 0.6                      py36_0    defaults
testpath                  0.3                      py36_0    defaults
theano                    0.9.0                    py36_0    defaults
tk                        8.5.18                        0    defaults
toolz                     0.8.2                     <pip>
toolz                     0.8.2                    py36_0    defaults
tornado                   4.4.2                    py36_0    defaults
traitlets                 4.3.2                    py36_0    defaults
wcwidth                   0.1.7                    py36_0    defaults
werkzeug                  0.12.2                   py36_0    defaults
wheel                     0.29.0                   py36_0    defaults
widgetsnbextension        3.1.4                    py36_0    conda-forge
xarray                    0.9.2                    py36_0    defaults
xz                        5.2.2                         1    defaults
yaml                      0.1.6                         0    defaults
zeromq                    4.1.5                         0    defaults
zlib                      1.2.8                         3    defaults
```
</p></details>

**Comment by kgullikson88** [comment]
Hmm, that seems to have fixed it. I still get all the info messages, but the terminal and python are functional. Is there a quick explanation for why that was happening? Should I be periodically updating the python in my base env?

**Comment by jjhelmus** [comment]
tl;dr The conda provided compilers required a change to Python which must be present in the base environment.  Periodically updating Python in the base environment is a good idea.

A bit of background: Starting with the release of Anaconda 5.0, conda packages from Anaconda, Inc are compiled using up-to-date compilers for better performance and security.  The compilers themselves  are provided in conda packages.  The theano package requires these compilers as the library requires working compilers at run-time.

The messages are coming from the package which activates the compilers.  Setting **CC**, **CFLAGS**, etc are needed to use these compilers but can be a bit of a surprise. We added the messages to provide a heads up on then modification to the environmental variables.

Python stores the configuration of the build environment (compiler paths, CFLAGS, etc) when it is compiled.  These configuration are then reused when Python builds a C extension.  This creates a bit of a problem, Python is built with these new compilers, but they may not be available.  In such cases  building C extensions would fail.  

To accommodate this, the Python in Anaconda uses a generic set of build configuration by default which will work with most system provide compilers.  If the conda provided compilers are installed, which we do encourage if you are going to build C extensions, then the configuration for these compilers is used.  This switch if controlled by setting the **_PYTHON_SYSCONFIGDATA_NAME** environment variable.  

This behavior has worked well but there is one downside. Python's that were built before the switch to these new compilers do not include the logic required to make this switch and will error out if **_PYTHON_SYSCONFIG_DATA_NAME** is set.  If such a Python is installed in the base conda environment you get an error like you saw.  

The fix is relatively easy, update the Python to a version which knows about the new compiles.  This also gives you any bug fixes and improvement that have been made to Python.  Periodically updating the Python in the base environment is a great idea. That said, we prefer to not make these kind of breaking changes and try to minimize the instances where they are needed.  Updating Python in the base environment is recommended but we try very had to not make it a requirement.  In this case it was the best option.

**Comment by shadowwalkersb** [comment]
> The fix is relatively easy, update the Python to a version which knows about the new compiles.

What is the first version of Python that knows about the new compilers?

**Comment by jjhelmus** [comment]
> What is the first version of Python that knows about the new compilers?

These changes were added in:
  * Python 2.7.14 build 15
  * Python 3.5.4 build 14
  * Python 3.6.2 build 14
 
All later builds and versions should contain the needed patch.  Additionally, Python 2.7.13 build 15 and beyond contain the fix.

**Comment by shadowwalkersb** [comment]
Thanks, @jjhelmus !
